package.json

```diff
@@ -43,9 +44,11 @@
     "lerna-changelog": "^1.0.1",
     "prettier": "2.2.1",
     "react-is": "^17.0.1",
+    "react-pagination": "^1.0.0",
```

Comment:
@veljkocukic this is a local package from the mono-repo, what is this doing here?